
Thanks to Michael Stahl for the pointer (on the mozilla.dev.builds google group)
https://bugzilla.mozilla.org/show_bug.cgi?id=1298459

# HG changeset patch
# User Ryan VanderMeulen <ryanvm@gmail.com>
# Parent  a65b35c8e5b17c2585968974aef1da67a8c56642
Bug 1298459 - Only include AccessibleNode.webidl if accessibility is enabled.

diff --git a/dom/webidl/Node.webidl b/dom/webidl/Node.webidl
--- a/dom/webidl/Node.webidl
+++ b/dom/webidl/Node.webidl
@@ -103,11 +103,13 @@ interface Node : EventTarget {
   any getUserData(DOMString key);
   [ChromeOnly]
   readonly attribute Principal nodePrincipal;
   [ChromeOnly]
   readonly attribute URI? baseURIObject;
   [ChromeOnly]
   sequence<MutationObserver> getBoundMutationObservers();
 
+#ifdef ACCESSIBILITY
   [Pref="accessibility.AOM.enabled"]
   readonly attribute AccessibleNode? accessibleNode;
+#endif
 };
diff -r d0830980ffdb dom/webidl/moz.build
--- a/dom/webidl/moz.build	Fri Sep 02 13:32:41 2016 -0700
+++ b/dom/webidl/moz.build	Sat Sep 03 08:00:47 2016 +0200
@@ -11,6 +11,7 @@
 PREPROCESSED_WEBIDL_FILES = [
     'HTMLMediaElement.webidl',
     'Navigator.webidl',
+    'Node.webidl',
     'Promise.webidl',
     'PromiseDebugging.webidl',
     'Window.webidl',
@@ -18,7 +19,7 @@
 
 WEBIDL_FILES = [
     'AbstractWorker.webidl',
-    'AccessibleNode.webidl',
+#    'AccessibleNode.webidl',
     'AddonManager.webidl',
     'AnalyserNode.webidl',
     'Animatable.webidl',
@@ -350,7 +351,6 @@
     'NetDashboard.webidl',
     'NetworkInformation.webidl',
     'NetworkOptions.webidl',
-    'Node.webidl',
     'NodeFilter.webidl',
     'NodeIterator.webidl',
     'NodeList.webidl',
@@ -925,3 +925,8 @@
     WEBIDL_FILES += [
         'AlarmsManager.webidl',
     ]
+
+if CONFIG['ACCESSIBILITY']:
+    WEBIDL_FILES += [
+        'AccessibleNode.webidl',
+    ]
